Scripts/NuGet/EdFi.RestApi.Databases/prep-package-cloud.ps1

```diff
@@ -23,6 +23,8 @@ param(
     $outputDirectory
 )
 
+Write-Warning "'Scripts\NuGet\EdFi.RestApi.Databases\prep-package-cloud.ps1' is deprecated, and will be removed in the near future. Functionality is now included in 'logistics\scripts\activities\build\CloudOds\PrepareDatabasesForExport.ps1' instead."
```

Comment:
More style than anything, this warning uses back-slashes, while others use forward slashes. If we care for this deprecation warning, then you should change it, but I'm not sure we do.

Reply:
I've updated all these warning to make them more consistent.